ops/aws/canary/lib/canary-stack.ts

```diff
@@ -40,6 +40,7 @@ export class CanaryStack extends cdk.Stack {
         this.createLambdaScenarioFunc({action: "submitAndGet", timeoutMinutes: 1, rateMinutes: 2, memorySize: 1024});
         this.createLambdaScenarioFunc({action: "submitAndDescribe", timeoutMinutes: 1, rateMinutes: 2, memorySize: 256});
         this.createLambdaScenarioFunc({action: "submitWithConcurrency", timeoutMinutes: 1, rateMinutes: 2, memorySize: 256});
+        this.createLambdaScenarioFunc({action: "submitDockerIPFSJobAndGet", timeoutMinutes: 1, rateMinutes: 2, memorySize: 256});
```

Comment:
What I found out when running `submitAndGet` is that more memory is required, or else reading from IPFS sometimes hangs with no explanation or logs, and the scenario fails eventually after 1 minute timeout. Thats the reason submitAndGet uses 1GB to be stable, while other scenarios are okay with just 256MB. 

This scenario will need a minimum of 1GB as well, and might require more as it is downloading  and uploading a larger file compared to submitAndGet. We can start with 1GB and increase it later if we see issues.

Other than that, the PR looks good to me :)